e2e/api/steps.js

```diff
@@ -188,7 +196,10 @@ const assertValidData = async (data, pageId) => {
   const responseBody = await response.json();
 
   assert.equal(response.status, 200);
-  addIssueClaimMidEventFields(pageId, responseBody);
+
+  if (pagesWithMidEvents.includes(pageId)) {
```

Comment:
`midEventFieldForPage.hasOwnProperty(pageId)` could be used here, so we don't need `pagesWithMidEvents`